models/yang/sonic/sonic-ifa.yang

```diff
@@ -0,0 +1,126 @@
+module sonic-ifa {
+    namespace "http://github.com/Azure/sonic-ifa";
+    prefix ifa;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-acl {
+        prefix sacl;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC TAM IFA";
+
+    revision 2019-05-15 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-ifa {
+
+        container TAM_INT_IFA_FEATURE_TABLE {
+
+            list TAM_INT_IFA_FEATURE_TABLE_LIST {
+                key "feature";
+                max-elements "1";
+
+                leaf feature {
+                    type string;
+                    default "feature";
+                }
+
+                leaf enable {
+                    type boolean;
+                }
+            }
+        }
+
+        container TAM_DEVICE_TABLE {
+
+            list TAM_DEVICE_TABLE_LIST {
+                key "device";
+                max-elements "1";
+
+                leaf device {
+                    type string;
+                    default "device";
+                }
+
+                leaf deviceid {
+                    type uint16;
+                    default 0;
+                }
+            }
+        }
+
+        container TAM_COLLECTOR_TABLE {
```

Comment:
@spenugondaa : Please move them to separate sonic-tam.yang

Reply:
Done